pkg/serviceactivities/shutdown/shutdown.go

```diff
@@ -62,3 +73,22 @@ func (s *ServiceActivity) Close(_ context.Context) error {
 	close(s.done)
 	return nil
 }
+
+// HandleShutdownConditions encapsulates the shutdown logging logic for services into a simple function, and returns
+// a boolean indicating if it was a graceful shutdown or not
```

Comment:
It's not really _was_

```suggestion
// HandleShutdownConditions encapsulates the shutdown logging logic for
// services into a simple function, and returns a boolean indicating if
// a graceful shutdown has started or not.
```